src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/WrappingTable.cs

```diff
@@ -0,0 +1,235 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal class Column
+    {
+        public string Header { get; set; }
+        public int Width { get; set; }
+        public bool Highlight { get; set; }
+    }
+
+    internal class WrappingTable
```

Comment:
you probably don't need the table to be of a different type. 
We only have 1 table, so keeping the type name would minimize the changes in other files.